api/src/shop/transactions.py

```diff
@@ -62,9 +74,6 @@ def commit_transaction_to_db(member_id=None, total_amount=None, contents=None, s
         # Mark this transaction as one that is for registering a member.
         db_session.add(PendingRegistration(transaction_id=transaction.id))
 
-    # TODO Rename stripe pending to TransactionReferences or something (with type).
-    db_session.add(StripePending(transaction_id=transaction.id, stripe_token=stripe_card_source_id))
```

Comment:
Vi borde nog inte flytta db_session.add(StripePending(transaction_id=transaction.id,  stripe_token=stripe_card_source_id)) utanför @nested_atomic tror jag. Risken med det är att vi får en transaktion i vår db som inte har någon extern referens och det är ju tråkigt när vi har referensen tillgänglig i requestet.

Även om den har det lustiga namnet StripePending så har det egentligen inte så mycket med stripe att göra. Det är helt enkelt en extern referens till transaktionen. I testet går det ju bra att skicka in vilken slumsträng som helst.